warehouse/query-core/src/main/java/datawave/query/tables/ShardIndexQueryTable.java

```diff
@@ -250,6 +250,10 @@ public GenericQueryConfiguration initialize(Connector connection, Query settings
         ASTJexlScript script;
         try {
             Set<String> expansionFields = metadataHelper.getExpansionFields(config.getDatatypeFilter());
+            if (expansionFields.isEmpty()) {
```

Comment:
This is not needed because we pull the fields down in the `ShardIndexQueryTableStaticMethods.getIndexedExpansionFields(..)` method and may be a bug.

Reply:
good find!  Thank you @drewfarris